app/filtering.js

```diff
@@ -198,32 +204,25 @@ function registerPermissionHandler (session) {
   // TODO: Localize strings
   let permissions = {
     media: {
-      action: 'use your camera and/or microphone',
-      hosts: {}
+      action: 'use your camera and/or microphone'
```

Comment:
You don't have to now but this will require a bit of localization work. The strings should go into locale.js and also the string action shouldn't be a variable.  Only host would be a variable.  The action might move around in the sentence in different languages.